plugin/externalPluginEntry.go

```diff
@@ -15,6 +15,9 @@ import (
 	"github.com/puppetlabs/wash/journal"
 )
 
+/*
+TODO: Come back and rethink attributes once all the attributes/metadata work is finished.
```

Comment:
I'll probably do this much later, after we've got some use-cases addressed. The main fixes would likely be to beef-up the attributes key w/ more attributes + add a separate `synced_attributes` key.